manifests/server/config.pp

```diff
@@ -35,6 +35,13 @@
     mode   => '0700',
   }
 
+  file { $mcollective::ssldir:
```

Comment:
@petems i honestly think there's no harm in just having that resource declaration, as-is.